app/helpers/schools_helper.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module SchoolsHelper
+  def school_display_name(school, duplicate_schools = [], show_postcode: false)
```

Comment:
```suggestion
  def location_display_name(school, duplicate_schools = [], show_postcode: false)
```

Since I don't think there's anything specific about schools here, and in theory when we have community clinics they could also have duplicate names?

For community clinic we would show ODS code instead of URN.